exercises/practice/yacht/Yacht.ps1

```diff
@@ -0,0 +1,44 @@
+# Score categories. Change the values as you see fit.
+Enum Category {
+    YACHT           = $null
+    ONES            = $null
+    TWOS            = $null
+    THREES          = $null
+    FOURS           = $null
+    FIVES           = $null
+    SIXES           = $null
+    FULL_HOUSE      = $null
+    FOUR_OF_A_KIND  = $null
+    LITTLE_STRAIGHT = $null
+    BIG_STRAIGHT    = $null
+    CHOICE          = $null
+}
+
+Function Get-Score() {
+    <#
+    .SYNOPSIS
+    Implement a function to get the score of a yatch game.
+
+    .DESCRIPTION
+    Given a list of values for five dice and a category, your solution should return the score of the dice for that category.
+    If the dice do not satisfy the requirements of the category your solution should return 0.
+    You can assume that five values will always be presented, and the value of each will be between one and six inclusively.
+    You should not assume that the dice are ordered.
+
+    .PARAMETER Dice
+    An array of 5 integer, each represent a dice value.
+
+    .PARAMETER Category
+    An Enum value represent a category in the game of yatch.
```

Comment:
```suggestion
    An Enum value represent a category in the game of yacht.
```

Reply:
Ahh, I thought I catch all the typos. Thanks.